app/build.gradle

```diff
@@ -102,6 +102,7 @@ dependencies {
     implementation 'com.github.vbmacher:java-cup-runtime:11b-20160615-3'
     //noinspection GradleDependency
     implementation 'com.ibm.icu:icu4j:60.2'
+    implementation 'androidx.test.ext:junit:1.2.1'
```

Comment:
I think if we added this in `implementation`, others can be removed since they testImplementation/androidTestImplementation inherit the `implementation`

Reply:
removed